testdata/networking/generic_test_pod_with_replica.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
```

Comment:
do we still need to this file? remembered we have one list_for_pods.json

Reply:
let me take a look. I guess we might be able to reuse that. The last time i tried is it creates a service as well and when we tries to re-create pods, it sborts with error sayign svc already exist.

Reply:
aha, get it. there is a service in list_for_pods.json.   then it's not a problem